packages/x-components/src/x-modules/scroll/components/main-scroll.vue

```diff
@@ -1,19 +1,16 @@
 <template>
-  <NoElement :class="dynamicClasses">
+  <div ref="rootRef" :class="dynamicClasses">
```

Comment:
Styling this `div` in the component itself could be an option? In this way, we could avoid breaking changes in this PR

Reply:
I think that highly depends on the layout implementation from the outside. Not sure if we could just style it here